.circleci/config.yml

```diff
@@ -511,16 +511,15 @@ workflows:
       - check-breaking-change:
           requires:
             - prefetch-crates
-
   ######################################################################################################################
   # Used by bors to test prs on master's head before committing to master.                                             #
   # The "commit-workflow" must support auto and canary branchs...                                                      #
   # The "commit-workflow", and the  pull-request-workflow should be identical.                                         #
   ######################################################################################################################
-  commit-workflow:
-    jobs:
-      - docker-pre-publish:
-          context: docker
-          filters:
-            branches:
-              only: auto
+  #commit-workflow:
+  #  jobs:
+  #    - docker-pre-publish:
+  #        context: docker
+  #        filters:
+  #          branches:
+  #            only: auto
```

Comment:
Bors is still looking for a "commit-workflow" status so we'll need some job here still (till I do another release of bors to stop depending on it)

Reply:
updated to keep the package pull.   once this is merged in to master another pr can disable both jobs.    I really do think bors should look at a flat file for which checks to honor.